README.md

```diff
@@ -33,8 +33,25 @@ Example:
 ```js
 import atomLinter from 'atom-linter'
 
-const myLinter = {
+const myFileLinter = {
   // ...
+  scope: 'file',
+  async lint(textEditor) {
+    const textEditorPath = textEditor.getPath()
+    const output = atomLinter.exec('myprogram', ['parameter1', 'parameter2'], { uniqueKey: `my-linter:${textEditorPath}` })
```

Comment:
If people are copy pasting, they would start by copy/pasting the example in `linter`, not here. Plus the fact that it didn't work might give them a clue that they had done something wrong 😛.